packages/vm/src/compatibility.rs

```diff
@@ -255,6 +222,7 @@ fn check_wasm_exports(module: &ParsedWasm, logs: Logger) -> VmResult<()> {
 /// or a error in the contract.
 fn check_wasm_imports(
     module: &ParsedWasm,
+    limits: &WasmLimits,
     supported_imports: &[&str],
```

Comment:
Should we move limits to the 3rd place to always have it behind the primary input? In `check_wasm_capabilities(` we have it like that too